dotnet/src/Agents/UnitTests/Agents.UnitTests.csproj

```
@@ -54,7 +54,8 @@
 
   <ItemGroup>
     <Compile Include="$(RepoRoot)/dotnet/src/InternalUtilities/test/AssertExtensions.cs" Link="%(RecursiveDir)Test/%(Filename)%(Extension)" />
-    <Compile Include="$(RepoRoot)/dotnet/src/InternalUtilities/test/HttpMessageHandlerStub.cs" Link="%(RecursiveDir)Test/%(Filename)%(Extension)" />
+    <Compile Include="$(RepoRoot)/dotnet/src/InternalUtilities/test/HttpMessageHandlerStub.cs" Link="%(RecursiveDir)Test/%(Filename)%(Extension)" />    
+    <Compile Include="$(RepoRoot)/dotnet/src/InternalUtilities/test/MultipleHttpMessageHandlerStub.cs" Link="%(RecursiveDir)Test/%(Filename)%(Extension)" />
```

nit.

`MultipleHttpMessageHandlerStub` has expanded capabilities of `HttpMessageHandlerStub`, when using the multiple in new code I don't think we need to add the `HttpMessageHandlerStub` also.

```suggestion
    <Compile Include="$(RepoRoot)/dotnet/src/InternalUtilities/test/MultipleHttpMessageHandlerStub.cs" Link="%(RecursiveDir)Test/%(Filename)%(Extension)" />
```

---

My new tests all use `MultipleHttpMessageHandlerStub` however there are existing tests which rely on `HttpMessageHandlerStub`. I've created a task to track this change: https://github.com/microsoft/semantic-kernel/issues/12431